src/resources/views/crud/fields/checklist.blade.php

```diff
@@ -61,7 +69,7 @@
             function bpFieldInitChecklist(element) {
                 var hidden_input = element.find('input[type=hidden]');
                 var selected_options = JSON.parse(hidden_input.val() || '[]');
-                var checkboxes = element.find('input[type=checkbox]');
+                var checkboxes = element.find('#checkbox-wrapper :input[type=checkbox]');
```

Comment:
This is what I mean. If the "select all" was an anchor instead of a checkbox... this change wouldn't have been needed 😀 Not 100% sure anchor is a good idea... no idea how this looks. But yeah, just saying.